list.html

```diff
@@ -0,0 +1,81 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet" type="text/css" href="application.css">
+</head>
+
+<body class="body">
+  <div class="books-container row">
+    <div class="book column shadow-container">
+      <div class="book-info">
+        <img src="./assets/book-cover.png" alt="Book cover" class="book-cover-list">
+        <h4 class="book-list-title text-3">Título del libro</h4>
+        <p class="book-list-author text-4">Autor del libro</p>
+      </div>
+    </div>
+
+    <div class="book column shadow-container">
+      <div class="book-info">
+        <img src="./assets/book-cover.png" alt="Book cover" class="book-cover-list">
```

Comment:
I think that "./" is not necessary. But I'm not 100% sure.

Reply:
Yes, you are right. This is not needed "./"